release/downloadIstioCandidate.sh

```diff
@@ -109,7 +109,7 @@ ARCH_UNSUPPORTED="1.5"
 
 if [ "${OS}" = "Linux" ] ; then
   # This checks if 1.6 <= 1.5 or 1.4 <= 1.5
-  if [ "$(expr "${ARCH_SUPPORTED}" \<= "${ARCH_UNSUPPORTED}")" -eq 1 ]; then
+  if awk 'BEGIN {exit !('$ARCH_SUPPORTED' <= '$ARCH_UNSUPPORTED')}'; then
```

Comment:
@ericvn After a couple of tests, I'm afraid this is still not comparing float value properly. It only compares floating-point numbers if both have the same decimal points. so `1.5 >= 1.6` pass but `1.5 >= 1.10` fails. Any better idea?

Reply:
A version compare is more than a few lines if in bash only. I wonder if we could assume there was a `sort` command that supported version sort. Makes it easier, but might not work everywhere.

Reply:
It seems a number of tools who used to download a script and run it have mostly changed to just having one download the actual binary. I believe `kubectl` used to run a script in the past and has moved to just getting the actual download.

Reply:
> A version compare is more than a few lines if in bash only. I wonder if we could assume there was a sort command that supported version sort. Makes it easier, but might not work everywhere.

I have tried few options that work in `bash` but not in `shell` (bash seems easier). 

Wondering if we still support download lower than 1.5 (without arch support)? that is the problem.

>  I believe kubectl used to run a script in the past and has moved to just getting the actual download.

Maybe we can also have multiple conditions for different download options. For 1.5 and lower which will be used rarely as compared to versions greater than 1.6 and above.

Reply:
@ericvn I applied a hack for floating-point comparison with 3 digits and now it's working as expected. Let me know if this looks good.